MovingBlocks/SplashScreen
# build: bump gradle 8.2 -> 8.2.1

Relates to https://github.com/MovingBlocks/Terasology/pull/5109
Relates to https://github.com/MovingBlocks/Terasology/pull/4653

### Contains

* Updating gradle from 8.2 -> 8.2.1

### How to test

1. Go into your Terasology workspace
2. Run `groovyw lib get SplashScreen`
3. In `libs/SplashScreen` check out this branch
4. Start Terasology
5. Check that the SplashScreen works fine

### Remarks

Java 20 is not yet supported, see https://github.com/gradle/gradle/issues/23488.
